src/features/dotnetTest.ts

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import {OmnisharpServer} from '../omnisharpServer';
+import {toRange} from '../typeConvertion';
+import * as vscode from 'vscode';
+import * as serverUtils from "../omnisharpUtils";
+import * as protocol from '../protocol';
+
+let enableDebug = false;
+
+// check if debugger start is enable
+vscode.commands.getCommands().then(commands => {
```

Comment:
I wonder if we want to make this part of the first call to updateCodeLensForTest rather than having this in the global scope. Otherwise it seems like we have a bit of non-determinism here.